kfs-core/src/main/resources/edu/arizona/kfs/fp/document/datadictionary/CashReceiptDocument.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+	xmlns:dd="http://rice.kuali.org/dd"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd         http://rice.kuali.org/dd         http://rice.kuali.org/dd/dd.xsd">
+	
+	<!-- workflow attributes for routing -->
+	<bean id="CashReceiptDocument-workflowAttributes" parent="CashReceiptDocument-workflowAttributes-parentBean">
+	  <property name="routingTypeDefinitions">
+        <map>
+			<entry key="CashManagement" value-ref="CashReceiptDocument-RoutingType-CashReceiptCampus"/>
+			<entry key="ObjectSubTypeCode" value-ref="RoutingType-AccountingDocument-ObjectSubTypeCode"/>
```

Comment:
Misty has confirmed that the ObjectSubTypeCode routing should be before the CashManagement routing.